baksnapperd.sh

```diff
@@ -38,6 +38,21 @@ function warning {
     echo -e "[Warning] $1" 1>&2
 }
 
+function link-latest {
+    arr=()
+    for dir in "$1"/*; do
+        if ! [[ $(echo "$dir" | grep "latest") ]]; then
```

Comment:
No need to shell out to just check if latest is in the backup directory. Also there is a subtile bug if the backup location for some reason contains latest e.g. `$1 == /my/latest/backup/`.
Instead you can do this with [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) see the section about ${parameter##word}
I.e.
```
if  [[ "${dir##*/}" != "latest" ]]
```